src/main/java/core/basesyntax/FibonacciSequence.java

```diff
@@ -9,7 +9,21 @@
      * <p>Пример: n = 8
      * Результат: 21</p>
      */
+
     public int getFibonacciNumber(int index) {
-        return 0;
+        if (index == 0) {
+            return 0;
+        }
+
+        if (index == 1) {
+            return 1;
+        }
+
+        int result = 1;
```

Comment:
variable `result` is redundant. Consider removing it

Reply:
Done